recipe/meta.yaml

```diff
@@ -199,21 +170,21 @@ outputs:
         - binutils_impl_{{ target_platform }}  # [linux]
 {% endif %}
       host:
-        - bzip2  # [not win]
+        - bzip2
         - sqlite
-        - xz  # [not win]
-        - zlib  # [not win]
+        - xz
+        - zlib
         - openssl
         - readline  # [not win]
-        - tk  # [not win]
+        - tk
         - ncurses  # [unix]
-        - libffi   # [not (win or (osx and arm64))]
+        - libffi
         - ld_impl_{{ target_platform }} >=2.36.1  # [linux]
+        - libuuid  # [unix]
```

Comment:
Do we want `- libuuid  # [linux]` here and link to `libSystem.dylib` (or wherever the libuuid symbols live on macOS)? This is regarding https://github.com/conda-forge/libuuid-feedstock/issues/16 .
(I'm only aware of the above issue but lack knowledge on the details so can't say which way is better one, sorry.)

Reply:
Going off of https://github.com/regro/cf-scripts/issues/417 and the linked issues, IIUC, at least one problem is that if `libuuid` is pulled into an environment, then `include/uuid/uuid.h` could conflict with the one provided by the macOS SDK. I.e, this would more be a downstream issue than for `python` itself.
Maybe @chrisburr know more and could comment if pulling in `libuuid` via `python` could lead to problems?

Reply:
> at least one problem is that if libuuid is pulled into an environment, then include/uuid/uuid.h could conflict with the one provided by the macOS SDK.

How would it conflict though?

Reply:
nvm, saw the conversation in root